diff --git a/security/baseband_guard/Kconfig b/security/baseband_guard/Kconfig
new file mode 100644
index 0000000..2222222
--- /dev/null
+++ b/security/baseband_guard/Kconfig
@@ -0,0 +1,15 @@
+config BASEBAND_GUARD
+    bool "Baseband Guard"
+    default n
+    help
+      This LSM protects baseband partitions from write access.
+
+if BASEBAND_GUARD
+
+config BASEBAND_GUARD_SYSFS
+    bool "Sysfs config for baseband_guard"
+    default y
+
+endif  # BASEBAND_GUARD
